service/topology_state_machine.hh

```diff
@@ -47,6 +47,10 @@ enum class topology_request: uint8_t {
 
 using request_param = std::variant<raft::server_id, sstring, uint32_t>;
 
+enum class global_topology_request: uint8_t {
```

Comment:
For some states like `node_state` it may be enough to have 8 bits since if we will have a state machine with more then 256 states we will probably go crazy. But I may easily see more then 256 kinds of supported requests if some will be deprecated with time. Anyway I guess there is no harm to make all of them the same.